### PR TITLE
Fix states selector in back office stores form

### PR DESCRIPTION
**Pull request by zuk3975**
…point for choices

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | States controller was migrated and deleted, and apparently one ajax call in admin.js is not working anymore. You can see more in related issue (https://github.com/PrestaShop/PrestaShop/issues/32316) and initial PR (https://github.com/PrestaShop/PrestaShop/pull/32550). As stores page is being migrated and might will be finished for v9.0 (or later), this is just a quickfix for old page to still work.
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | #32316
| Fixed ticket?     | #32316
| Related PRs       | If theme, autoupgrade or other module change is needed to make this change work, provide a link to related PRs here.
| Sponsor company   | Your company or customer's name goes here (if applicable).

### CODE REVIEW

**Comment by kpodemski** [comment]
@zuk3975 I think you can target `8.1.x`, it is a safe fix

**Comment by zuk3975** [comment]
> @zuk3975 I think you can target `8.1.x`, it is a safe fix

I just checked, in 8.1.x the AdminStatesController still exists, so this fix is ok to target develop :heavy_check_mark: (sorry for misleading comment/question in description, I deleted it now :smile: )

File: admin-dev/themes/default/template/helpers/form/form.tpl [hunk 1]
```diff
@@ -67,7 +67,7 @@
 					<div class="form-wrapper">
 					{foreach $field as $input}
 						{block name="input_row"}
-						<div class="form-group{if isset($input.form_group_class)} {$input.form_group_class}{/if}{if $input.type == 'hidden'} hide{/if}"{if $input.name == 'id_state'} id="contains_states"{if !$contains_states} style="display:none;"{/if}{/if}{if $input.name == 'dni'} id="dni_required"{if !$dni_required} style="display:none;"{/if}{/if}{if isset($tabs) && isset($input.tab)} data-tab-id="{$input.tab}"{/if}>
+						<div class="form-group{if isset($input.form_group_class)} {$input.form_group_class}{/if}{if $input.type == 'hidden'} hide{/if}"{if $input.name == 'id_state'} id="contains_states" data-states-url={$states_url} {if !$contains_states} style="display:none;"{/if}{/if}{if $input.name == 'dni'} id="dni_required"{if !$dni_required} style="display:none;"{/if}{/if}{if isset($tabs) && isset($input.tab)} data-tab-id="{$input.tab}"{/if}>
```
Comment (kpodemski) [inline admin-dev/themes/default/template/helpers/form/form.tpl:70]: out of scope of course, but it reminds me how terrible things are still in the codebase, global form.tpl and if for the states page 🤕

**Review by kpodemski — APPROVED** [review]

**Review by tleon — APPROVED** [review]

**Review by Hlavtox — APPROVED** [review]

**Review by florine2623 — APPROVED** [review]
Hello @zuk3975 ,

Link to automated tests : https://github.com/florine2623/testing_pr/actions/runs/5000564715

It is working as expected, thanks for the PR !

https://github.com/PrestaShop/PrestaShop/assets/16019289/3035873b-b395-42a3-a5ab-193f80a4b118

It is QA ✅

**Pull request merged**